assets/aws/files/install.sh

```diff
@@ -62,7 +62,7 @@ if [ -f /tmp/teleport-fips ]; then
         mv ${TARBALL_FILENAME} /tmp/teleport.tar.gz
     else
         echo "Installing Enterprise Teleport version ${TELEPORT_VERSION} with FIPS support"
-        curl ${CURL_OPTS} -o teleport.tar.gz https://get.gravitational.com/teleport/${TELEPORT_VERSION}/teleport-ent-v${TELEPORT_VERSION}-linux-amd64-fips-bin.tar.gz
+        curl ${CURL_OPTS} -o teleport.tar.gz https://asset.idemeumlab.com/teleport/${TELEPORT_VERSION}/teleport-ent-v${TELEPORT_VERSION}-linux-amd64-fips-bin.tar.gz
```

Comment:
Use CDN Url property as it will be diff on staging vs production

Reply:
Only staging build.